docs/releases/2.0.txt

```diff
@@ -157,6 +157,10 @@ Forms
   HTML attributes for the ``DateInput`` and ``TimeInput`` (or hidden)
   subwidgets.
 
+* :class:`~django.forms.FloatField` and :class:`~django.forms.DecimalField`
```

Comment:
This looks more like a bug fix than a "minor feature" so we wouldn't have a release note for that typically.